Data/symbol-properties/symbol-properties-database.csv

```diff
@@ -349,543 +349,397 @@ oanda,USDTRY,forex,USD/TRY,TRY,1,0.00001,1
-bitfinex,WTCUSD,crypto,Walton-US Dollar,USD,1,0.00001,8.
-bitfinex,XAUTAUT,crypto,XAUT-AUT,AUT,1,0.00001,0.00000001
-bitfinex,XAUTBTC,crypto,XAUT-Bitcoin,BTC,1,0.00001,0.00000001
-bitfinex,XAUTUSD,crypto,XAUT-US Dollar,USD,1,0.00001,0.00000001
-bitfinex,XAUTUST,crypto,XAUT-Tether,UST,1,0.00001,0.00000001
-bitfinex,XCHETH,crypto,ClearingHouse-Ethereum,ETH,1,0.00001,2.
-bitfinex,XCHUSD,crypto,ClearingHouse-US Dollar,USD,1,0.00001,2.
-bitfinex,XLMBTC,crypto,Stellar Lumens-Bitcoin,BTC,1,0.00001,80.
-bitfinex,XLMETH,crypto,Stellar Lumens-Ethereum,ETH,1,0.00001,80.
-bitfinex,XLMEUR,crypto,Stellar Lumens-Euro,EUR,1,0.00001,80.
-bitfinex,XLMGBP,crypto,Stellar Lumens-Pound Sterling,GBP,1,0.00001,80.
-bitfinex,XLMJPY,crypto,Stellar Lumens-Yen,JPY,1,0.00001,0.00000001
-bitfinex,XLMUSD,crypto,Stellar Lumens-US Dollar,USD,1,0.00001,80.
-bitfinex,XMRBTC,crypto,Monero-Bitcoin,BTC,1,0.00001,0.08
-bitfinex,XMRUSD,crypto,Monero-US Dollar,USD,1,0.00001,0.08
-bitfinex,XRAETH,crypto,Ratecoin-Ethereum,ETH,1,0.00001,0.00000001
-bitfinex,XRAUSD,crypto,Ratecoin-US Dollar,USD,1,0.00001,472.
-bitfinex,XRPBTC,crypto,Ripple-Bitcoin,BTC,1,0.00001,20.
-bitfinex,XRPUSD,crypto,Ripple-US Dollar,USD,1,0.00001,20.
-bitfinex,XTZBTC,crypto,Tezos (Pre-Launch)-Bitcoin,BTC,1,0.00001,2.
-bitfinex,XTZUSD,crypto,Tezos (Pre-Launch)-US Dollar,USD,1,0.00001,2.
-bitfinex,XVGBTC,crypto,Verge-Bitcoin,BTC,1,0.00001,1402.
-bitfinex,XVGETH,crypto,Verge-Ethereum,ETH,1,0.00001,0.00000001
-bitfinex,XVGEUR,crypto,Verge-Euro,EUR,1,0.00001,0.00000001
-bitfinex,XVGGBP,crypto,Verge-Pound Sterling,GBP,1,0.00001,0.00000001
-bitfinex,XVGJPY,crypto,Verge-Yen,JPY,1,0.00001,0.00000001
-bitfinex,XVGUSD,crypto,Verge-US Dollar,USD,1,0.00001,1402.
-bitfinex,YGGETH,crypto,YGG-Ethereum,ETH,1,0.00001,0.00000001
-bitfinex,YGGUSD,crypto,YGG-US Dollar,USD,1,0.00001,13854.
-bitfinex,YYWBTC,crypto,YOYOW-Bitcoin,BTC,1,0.00001,0.00000001
-bitfinex,YYWETH,crypto,YOYOW-Ethereum,ETH,1,0.00001,0.00000001
-bitfinex,YYWUSD,crypto,YOYOW-US Dollar,USD,1,0.00001,478.
-bitfinex,ZBTUSD,crypto,ZBT-US Dollar,USD,1,0.00001,14.
-bitfinex,ZBTUST,crypto,ZBT-Tether,UST,1,0.00001,0.00000001
-bitfinex,ZCNBTC,crypto,0chain-Bitcoin,BTC,1,0.00001,0.00000001
-bitfinex,ZCNETH,crypto,0chain-Ethereum,ETH,1,0.00001,0.00000001
-bitfinex,ZCNUSD,crypto,0chain-US Dollar,USD,1,0.00001,44.
-bitfinex,ZECBTC,crypto,Zcash-Bitcoin,BTC,1,0.00001,0.2
-bitfinex,ZECUSD,crypto,Zcash-US Dollar,USD,1,0.00001,0.2
-bitfinex,ZILBTC,crypto,Zilliqa-Bitcoin,BTC,1,0.00001,0.00000001
-bitfinex,ZILETH,crypto,Zilliqa-Ethereum,ETH,1,0.00001,0.00000001
-bitfinex,ZILUSD,crypto,Zilliqa-US Dollar,USD,1,0.00001,748.
-bitfinex,ZRXBTC,crypto,0x-Bitcoin,BTC,1,0.00001,20.
-bitfinex,ZRXDAI,crypto,0x-DAI (Multi-collateral),DAI,1,0.00001,0.00000001
-bitfinex,ZRXETH,crypto,0x-Ethereum,ETH,1,0.00001,20.
-bitfinex,ZRXUSD,crypto,0x-US Dollar,USD,1,0.00001,20.
+bitfinex,TESTAAATESTBBB,crypto,AAA-BBB,TESTBBB,1,0.00001,0.00000001,tAAABBB
```

Comment:
Not sure this should be here `TESTAAATESTBBB`?

Reply:
Agree, will remove it